ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -619,4 +624,26 @@ static void maybeCreateProcessingLogStream(
 
     commandQueue.enqueueCommand(configured.get());
   }
+
+  private static void maybeConfigureRocksDBConfigSetter(final KsqlConfig ksqlConfig) {
+    final Map<String, Object> streamsProps = ksqlConfig.getKsqlStreamConfigProps();
+    if (streamsProps.containsKey(StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG)) {
+      final Class<?> clazz =
+          (Class) streamsProps.get(StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG);
+
+      final Method method;
+      try {
+        method = clazz.getMethod("configure", Map.class);
+      } catch (NoSuchMethodException e) {
+        // Nothing to configure
+        return;
+      }
+
+      try {
+        method.invoke(null, ksqlConfig.originals());
+      } catch (IllegalAccessException | InvocationTargetException e) {
+        throw new RuntimeException("failed to configure class: " + clazz.getName(), e);
```

Comment:
Include the config name, i.e. `StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG` in the error.